MANIFEST.in

```diff
@@ -1,7 +1,13 @@
-recursive-include docs *
-recursive-include pyramid_jinja2 *
+graft pyramid_jinja2
+graft docs
+prune docs/_build
 
-recursive-exclude docs *.pyc
-recursive-exclude pyramid_jinja2 *.pyc
+include README.rst CHANGES.rst
+include CONTRIBUTORS.txt LICENSE.txt COPYRIGHT.txt
 
-include *.txt *.ini *.rst
+include .coveragerc .flake8 setup.cfg pyproject.toml
+include tox.ini rtd.txt
+graft .github
```

Comment:
Why should `.github` be included? I see that Pyramid grafts it, too, but it's only used for CI and not really useful for distribution on a developer's machine. It's fine to leave it, I'm just curious.

Reply:
Same comment than before, I reused pyramid's file. I agree with your point.

Reply:
I based my opinion on https://packaging.python.org/guides/using-manifest-in/. I don't think it is needed.

Reply:
This will cause problems with the `check-manifest` lint command but I'm not sure it's super important.

Reply:
Correct, I'm picking nits. We could try `prune .github`, the inverse of `graft`.

Reply:
It is grafted because we have historically in our source dists shipped everything that was checked into git. We continue to do the same with our source distributions now. Please do NOT remove the graft of the .github folder. It is part of our source code and it should be in the source distributable.